classify/trackprediction.py

```diff
@@ -13,6 +13,7 @@ def __init__(self, labels, model):
         self.labels = labels
         self.prediction_per_track = {}
         self.model = model
+        self.classify_time = None
```

Comment:
I find it a bit odd that the classify time is defined here but is set in another file.   It kind of breaks the Object orientated principles and makes the code quite hard to maintain.    You might have a good reason for it of course.  I can't tell!